File: revisiting_ctci/1_3_URLify.py
import unittest



def URL(s):
	""" take an input string and replace spaces with %20"""
	return s.replace(" ", "%20")


class TestURL(unittest.TestCase):
	def test_URLfunc(self):
		self.assertEqual(URL("this is the string"), "this%20is%20the%20string")


if __name__ == '__main__':
	unittest.main()

File: revisiting_ctci/1_3_URLify.go
package main


import(
	"fmt"
	"strings"
)

// take the string and replace all spaces with %20
func URL(s string) string {
	return strings.Replace(s," ", "%20", -1 )
}


func main(){

	t1 := "this is the string"

	fmt.Printf("words: %v\n",  t1)
	fmt.Printf("url: %v\n", URL(t1) )

}